games/tic_tac_toe.c

```diff
@@ -218,6 +220,38 @@ void doublemode()
     }
 }
 
+/**
+ * @brief Helper function of place_x,place_y function
+ *
+ * @param None
```

Comment:
No need to add this if there are no parameters.

Reply:
okay